Apps/APN/components/Dashboard.js

```diff
@@ -86,7 +86,10 @@ const Dashboard = ({ navigation }) => {
   const registerDeviceToken = () => {
     if (deviceToken !== null) {
       CustomerIO.registerDeviceToken(deviceToken);
-      alert('Device token registered successfully');
+      //alert('Device token registered successfully');
+
+      CustomerIO.deleteDeviceToken();
+      alert('Registered, and then deleted token');
```

Comment:
This is only to be able to test the function in the sample app. We can revert this change before merging.